probe-rs/src/probe/arm_debug_interface.rs

```diff
@@ -1751,22 +1897,11 @@ mod test {
             0x0,
             0x0,
         );
-        // Check CTRL
-        mock.add_jtag_response(PortType::DebugPort, 4, true, DapAcknowledge::Ok, 0, 0);
-        mock.add_jtag_response(PortType::DebugPort, 12, true, DapAcknowledge::Ok, 0, 0);
 
         // Expect a Write to the ABORT register.
         mock.add_jtag_abort();
 
         // Second try to write register.
-        mock.add_jtag_response(
```

Comment:
I guess I shouldn't delete these, I need to re-run the AP transaction after an abort?